projects/dex-ui/src/components/Swap/TokenInput.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { MouseEvent, useCallback, useRef, useState } from "react";
+import { Token, TokenValue } from "@beanstalk/sdk";
+import { FC } from "src/types";
+import styled from "styled-components";
+import { BasicInput } from "./BasicInput";
+import { TokenPicker } from "./TokenPicker";
+import { reactWhatChanged as RWC } from "src/utils/what-changed";
```

Comment:
Is this even being used? If not can we remove the whole thing?

Reply:
not anymore, will remove this import